.github/workflows/build.yml

```diff
@@ -3,29 +3,49 @@ name: miza-kinesis - build and publish
 on: [push]
 
 env:
-  NODE_VERSION: '12.13'
+  NODE_VERSION: '12.14'
 
 jobs:
-  build:
-    name: build
+  test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - '10'
+          - '12.13'
+          - '14'
```

Comment:
is `14` currently supported as an AWS Lambda Nodejs runtime?

Reply:
no, but it's the next long term support - that's why I think it's good to know ahead of time whenever there are any issues with support, wdys?